src/main/kotlin/graphics/scenery/CatmullRomSpline.kt

```diff
@@ -0,0 +1,71 @@
+package graphics.scenery.proteins
+
+import cleargl.GLVector
+import kotlin.math.pow
+
+/**
+ * This class offers the logic for creating a CatmuLL Rom Spline. This is essentially
+ * a spline going through control points. For more information see:
+ * https://en.wikipedia.org/wiki/Centripetal_Catmull–Rom_spline
+ * @param controlPoints the list of control points
+ * @param alpha determines the kind of Catmull Rom Spline, set in range of 0..1
```

Comment:
This is a bit unclear, what exactly does the parameter control, and why is the default 0.5?

Reply:
Also, in kdoc (as opposed to Javadoc), parameters can be referred to by their name via `[myParameter]`, and don't need the `@param` annotation.